module-types/Datacom/PSU-400-AC-F.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Datacom
+model: PSU 400 AC-F
+part_number: 820.8006.xx
+comments: (Do not use for new orders, migrate to PSU 600 AC-F) AC power supply with 100-240VAC and 50/60Hz input. Forced ventilation with airflow exiting
+  through the front panel of the power supply. Supports hot-swap and operation in redundant mode with another compatible PSU.
+power-ports:
+  - name: PSU
```

Comment:
Please update this name to use the `{module}` definition

Reply:
sorry, but I don't understand how it should be